### PR TITLE
[v6r18] Fixing bad merge, and pylints

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: TransformationSystem/Agent/TransformationCleaningAgent.py [hunk 1]
```diff
@@ -142,21 +142,11 @@ def execute( self ):
         if transDict[ 'Type' ] in self.dataManipTTypes:
           res = self.archiveTransformation( transDict['TransformationID'] )
           if not res['OK']:
-<<<<<<< Updated upstream
```
Comment (chaen) [inline TransformationSystem/Agent/TransformationCleaningAgent.py]: We could try to spot this with post commit/post merge hooks, or even in the CI tests

**Comment by chaen** [comment]
Review OK

File: TransformationSystem/Agent/TransformationCleaningAgent.py (hunk 1 above)
Comment (fstagni) [inline TransformationSystem/Agent/TransformationCleaningAgent.py]: I did spot in Jenkins.

File: TransformationSystem/scripts/dirac-production-runjoblocal.py [hunk 2]
```diff
@@ -48,41 +48,40 @@ def __runSystemDefaults(jobID, vo):
...
+
   tempdir = str(vo) + "job" + str(jobID) + "temp"
 
   try:
     os.mkdir(tempdir)
     if not sys.exc_info()[1][0]:
-      S_OK("Temporary directory created.")    
-    pass
-  except:    
+      S_OK("Temporary directory created.")
```
Comment (phicharp) [inline TransformationSystem/scripts/dirac-production-runjoblocal.py]: What does this line do?

Comment (fstagni) [inline TransformationSystem/scripts/dirac-production-runjoblocal.py]: Fixed using mkDir from DIRAC.Core.Utilities.File

File: TransformationSystem/scripts/dirac-production-runjoblocal.py [hunk 3]
```diff
@@ -48,41 +48,31 @@ def __runSystemDefaults(jobID, vo):
...
   tempdir = str(vo) + "job" + str(jobID) + "temp"
 
-  try:
-    os.mkdir(tempdir)
-    if not sys.exc_info()[1][0]:
-      S_OK("Temporary directory created.")    
-    pass
-  except:    
-    if sys.exc_info()[1][0] == 17:
-      S_OK("Temporary directory already exists.")
```
Comment (phicharp) [inline TransformationSystem/scripts/dirac-production-runjoblocal.py]: Still my question holds: this line does absolutely nothing... What is it meant to do?

**Pull request merged**